packit_service/sentry_integration.py

```diff
@@ -42,10 +44,13 @@ def configure_sentry(
         f"sqlalchemy_integration={sqlalchemy_integration}"
     )
 
+    if config.disable_sentry:
+        return
+
     secret_key = getenv("SENTRY_SECRET")
+
     if not secret_key:
         return
```

Comment:
Now that I see these two together, isn't having yet another config flag to disable Sentry superfluous, when not setting the secret would achieve the same result?

Maybe this issue could be solved by updating the docs/tools for local deployments.

Reply:
I'd say this is the right thing to do: if disable_sentry is False and the key is not set, we'd get an error right away - without the config option, we should get an error once we want to send an event to sentry, which is pretty late

Reply:
> I'd say this is the right thing to do: if disable_sentry is False and the key is not set, we'd get an error right away...

```
if not secret_key:
    return
```

I see  no error here, am I missing something?

Reply:
I actually wrote that sentence without looking at the code :D

@IceWreck could you please update the code by raising an exc if the secret_key is not set? Thank you.